apps/full-site-editing/full-site-editing-plugin/mailerlite/subscriber-popup.php

```diff
@@ -0,0 +1,128 @@
+<?php //phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+namespace A8C\FSE\Mailerlite;
+
+/**
+ * Mailerlite widget class
+ * Display a subscriber popup for Mailerlite.
+ */
+class WPCOM_Widget_Mailerlite extends \WP_Widget {
+
+	/**
+	 * WPCOM_Widget_Mailerlite constructor.
+	 */
+	public function __construct() {
+		parent::__construct(
+			'wpcom-mailerlite',
+			/** This filter is documented in modules/widgets/facebook-likebox.php */
+			apply_filters( 'jetpack_widget_name', __( 'Mailerlite subscriber popup', 'jetpack' ) ),
+			array(
+				'classname'                   => 'widget_mailerlite',
+				'description'                 => __( 'Display Mailerlite subscriber popup', 'jetpack' ),
+				'customize_selective_refresh' => true,
+			)
+		);
+	}
+
+	/**
+	 * Output the widget.
+	 *
+	 * @param array $args Display arguments including 'before_title', 'after_title', 'before_widget', and 'after_widget'.
+	 * @param array $instance - The settings for the particular instance of the widget.
+	 */
+	public function widget( $args, $instance ) {
+		/** This action is documented in modules/widgets/gravatar-profile.php */
+		do_action( 'jetpack_stats_extra', 'widget_view', 'mailerlite' );
+
+		if ( empty( $instance['account'] ) || empty( $instance['uuid'] ) ) {
+			if ( current_user_can( 'edit_theme_options' ) ) {
+				echo esc_js( $args['before_widget'] );
+				echo '<p>' . sprintf(
+					wp_kses(
+						/* translators: %1$s - URL to manage the widget, %2$s - documentation URL. */
+						__( 'You need to enter your numeric account ID and UUID for the <a href="%1$s">Mailerlite Widget</a> to work correctly. <a href="%2$s" target="_blank">Full instructions</a>.', 'jetpack' ),
+						array(
+							'a' => array(
+								'href'  => array(),
+								'title' => array(),
+							),
+						)
+					),
+					esc_url( admin_url( 'widgets.php' ) ),
+					'https://support.wordpress.com/widgets/mailerlite'
```

Comment:
This will just get forwarded to `https://wordpress.com/support/widgets/mailerlite`, so we could use it directly (the support forums were moved in the past few months I think). Noting it's also an empty page so far.

Reply:
Fixed!